etcdserver/api/v2http/capability.go

```diff
@@ -63,6 +66,9 @@ func capabilityLoop(s *etcdserver.EtcdServer) {
 			enableMapMu.Lock()
 			enabledMap = capabilityMaps[pv.String()]
 			enableMapMu.Unlock()
+			once.Do(func() {
```

Comment:
I don't think the `once` is necessary because of the `if v := s.ClusterVersion(); v != pv {` check